### PR TITLE
feat: support serving local files

### LINKED ISSUE #20: Support serving local files
```
.
├── index.html
└── mod.ts
```

`mod.ts`:
```ts
addEventListener("fetch", (event) => {
  event.respondWith(fetch(new URL("index.html", import.meta.url)));
});
```

```
deployctl run --no-check mod.ts 
```
Getting the following error when running the above command.

```
Listening on http://0.0.0.0:8080
error: Uncaught (in promise) TypeError: scheme 'file' not supported
  event.respondWith(fetch(new URL("index.html", import.meta.url)));
^
    at handleError (deno:core/core.js:186:12)
    at jsonOpParseResult (deno:core/core.js:356:9)
    at opSync (deno:core/core.js:259:32)
    at Object.jsonOpSync (deno:core/core.js:372:12)
    at opFetch (deno:op_crates/fetch/26_fetch.js:887:17)
    at sendFetchReq (deno:op_crates/fetch/26_fetch.js:1257:44)
    at fetch (deno:op_crates/fetch/26_fetch.js:1379:31)
    at file:///Users/sr/c/github.com/denoworld/deno.world/dir/mod.ts:8:1
    at innerInvokeEventListeners (deno:op_crates/web/02_event.js:750:27)
    at invokeEventListeners (deno:op_crates/web/02_event.js:785:5)
error: Process exited with code 1
```

### PR DESCRIPTION
**Pull request by satyarohith**
Introduces support to serve local files during local development.

Closes #20
Closes https://github.com/denoland/deploy_feedback/issues/3

### CODE REVIEW

File: src/runtime.js [hunk 1]
```diff
@@ -57,7 +58,6 @@ class FetchEvent extends Event {
         ? readerFromStreamReader(resp.body.getReader())
         : undefined,
     });
-    return resp;
```
Comment (kt3k) [inline src/runtime.js:60]: 👀 👍  ref: https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/respondWith#return_value

Comment (kt3k) [reply src/runtime.js:60]: ah, this still returns Promise<void>, but according to the spec this should return undefined. Maybe we should stop using async function here.

File: examples/serve_local.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
+addEventListener("fetch", (event) => {
+  event.respondWith(
+    fetch(new URL("../tests/testdata/data.json", import.meta.url)),
```
Comment (kt3k) [inline examples/serve_local.ts:4]: 👍

File: src/runtime.js (hunk 1 above)
Comment (satyarohith) [reply src/runtime.js:68]: We need it to be async to await the response. https://github.com/denoland/deployctl/blob/534d3dde87a95c7b3ee3c0990a9f24d9618b4d5a/src/runtime.js#L60

Comment (kt3k) [reply src/runtime.js:68]: I think we can use `response.then` instead of await, but let's do that in another branch.

**Review by kt3k — APPROVED** [review]
LGTM. Nice feature to have!

**Pull request merged**